### PR TITLE
docs: update manual install docs for v0.0.11 release

### LINKED ISSUE #53: Update to EncryptionConfiguration
https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/#encrypting-your-data-with-the-kms-provider

### LINKED ISSUE #72: add docs for running kms plugin as a static pod

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**Reason for Change**:

**Issue Fixed**:

fixes #53 
fixes #72 

**Notes for Reviewers**:

### CODE REVIEW

File: docs/manual-install.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+              path: /healthz
+              port: 8787
+            failureThreshold: 2
+            periodSeconds: 10
+          resources:
+            requests:
+              cpu: 100m
+              memory: 128Mi
+            limits:
+              cpu: 4
```
Comment (aramase) [inline docs/manual-install.md:71]: These are current defaults based on the manifests in aks-engine. We'll update this to a more concrete value after load testing: https://github.com/Azure/kubernetes-kms/issues/79

File: docs/manual-install.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    az group create -n $RG -l $LOC
+    # create keyvault
+    az keyvault create -n $KV_NAME -g $RG
+    # create key that will be used for encryption
+    az keyvault key create -n k8s --vault-name $KV_NAME --kty RSA --size 2048
+    ```
+
+1. Give the cluster identity permissions to access the keys in keyvault
+
+    The KMS Plugin uses the cluster service principal or managed identity to access the keyvault instance.
```
Comment (ritazh) [inline docs/manual-install.md:24]: You probably need to explain these identities a bit more

File: docs/manual-install.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+2. Create encryption configuration
+
+    Create a new encryption configuration file `/etc/kubernetes/manifests/encryptionconfig.yaml` using the appropriate properties for the `kms` provider:
+
+    ```yaml
+    kind: EncryptionConfiguration
+    apiVersion: apiserver.config.k8s.io/v1
+    resources:
+      - resources:
+          - secrets
```
Comment (ritazh) [inline docs/manual-install.md:115]: maybe add some comments for these are the resources api server will call the plugin to encrypt and decrypt

File: docs/manual-install.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        providers:
+          - kms:
+              name: azurekmsprovider
+              endpoint: unix:///opt/azurekms.socket
```
Comment (ritazh) [inline docs/manual-install.md:119]: maybe add a comment for how this endpoint needs to match the one created by the plugin which is configurable

File: docs/manual-install.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+              cachesize: 1000
+          - identity: {}
+    ```
+
+    The encryption configuration file needs to be created in all the master nodes.
```
Comment (ritazh) [inline docs/manual-install.md:124]: we should say as long as it's accessible by all the api servers. does not necessarily have to be on all the master nodes.

**Review by ritazh — APPROVED** [review]

**Pull request merged**